airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py

```diff
@@ -82,6 +98,12 @@ def transform_record(self, record: dict[str, Any], file: RemoteFile, last_update
         record[self.ab_file_name_col] = file.uri
         return record
 
+    def transform_record_for_file_transfer(self, record: dict[str, Any], file: RemoteFile) -> dict[str, Any]:
+        # timstamp() returns a float representing the number of seconds since the unix epoch
+        record[self.modified] = int(file.last_modified.timestamp()) * 1000
+        record[self.source_file_url] = file.uri
+        return record
```

Comment:
I ran into an issue where `RemoteFile.uri` is actually not a URI at all but is the relative path on the bucket.

Probably we aren't using this field for anything as of now but just want to call it out.

Reply:
Acknowledged, and yes, we are not currently using this, but "file_url" and "file_relative_path".